web_src/js/features/repo-diff.js

```diff
@@ -8,6 +8,24 @@ const {csrfToken} = window.config;
 export function initRepoDiffReviewButton() {
   $(document).on('click', 'button[name="is_review"]', (e) => {
     $(e.target).closest('form').append('<input type="hidden" name="is_review" value="true">');
+
+    // Watch for the form's submit event.
+    e.target.closest('form').addEventListener('submit', () => {
+      // Set a zero-timeout, so this would be executed after the network request has finished.
+      setTimeout(() => {
+        const commentCounter = document.querySelector('#review-box .review-comments-counter');
+        // Remove the display: none.
+        commentCounter.style.display = '';
+        // Increase counter by one, in case it's the first review, parseInt will return `NaN`
+        // and default back to '1'.
+        commentCounter.textContent = parseInt(commentCounter.textContent) + 1 || '1';
+
+        // Make the review-box to do a little pulse.
+        document.querySelector('#review-box').classList.add('pulse');
```

Comment:
can add a variable for `const reviewBox = document.querySelector('#review-box')` on top of the function to avoid multiple `querySelector` calls.